processes/development/new-app-setup/README.md

```diff
@@ -55,32 +55,35 @@ This assumes the discovery week has already happened.
 
 ## CI
 
-AWS config:
+#### AWS config:
 
 1. Create IAM user with read-only access to production S3 bucket and read/write access  to staging S3 bucket
     1. [Add policy](https://drive.google.com/open?id=0B4JoTt-NyIq5Y2RuYjZPTFAwd0U)
     2. Ensure the user has API credentials, no password
 
-MongoDB create users:
+#### MongoDB create users:
 
 1. Create read-only user for production DB
 2. Create user on staging DB (not read-only)
 
-Semaphore config:
+#### Semaphore config:
 
 1. Create new project
     1. NOTE: Paul needs to do this step currently. Will fix...
-1. Edit build settings:
+1. Edit Build Settings:
     1. Go to [https://semaphoreci.com/okgrow/](https://semaphoreci.com/okgrow/)&lt;PROJECT_NAME&gt;/settings
     1. Set the Node version to node.js 6.3.1
     1. Add each of these lines under setup
         1. `curl https://install.meteor.com/ | sh`
         1. `meteor --version`
-        1. `npm install`
-        1. `cp settings.json.example settings.json`
+        1. `meteor npm install`
+        1. `meteor npm start` _wip: find and add instructions to listen for success and exit process_
```

Comment:
we might need to use `meteor test --once` - but this would require a meteor integration test package, I think. https://guide.meteor.com/testing.html#command-line